Dockerfile

```diff
@@ -1,53 +1,78 @@
-FROM ubuntu:16.04
-RUN apt-get update && apt-get install -y \
-    git \
+FROM ubuntu:18.04
```

Comment:
There is some git bug that means that these are shown in the diff even though it's already been merged ...